src/registrar/management/commands/disclose_security_emails.py

```diff
@@ -0,0 +1,76 @@
+""""
+Converts all ready and DNS needed domains with a non-default public contact
+to disclose their public contact. Created for Issue#1535 to resolve
+ disclose issue of domains with missing security emails.
+"""
+
+import logging
+import copy
+
+from django.core.management import BaseCommand
+from registrar.models import Domain
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Disclose all nondefault domain security emails."
+
+    def __init__(self):
+        """Sets global variables for code tidyness"""
+        super().__init__()
+        # domains and transition domains that must be disclosed to true
+        self.contacts_saved: list[str] = []
+        # domains with errors, which are not successfully updated to disclose
+        self.domains_with_errors: list[str] = []
+        # domains that are successfully disclosed
+        self.disclosed_domain_contacts: list[str] = []
+        # domains that skip disclose due to having contact registrar@dotgov.gov
+        self.skipped_domain_contacts: list[str] = []
+
+    def handle(self, **options):
+        """
+        Converts all ready and DNS needed domains with a non-default public contact
+        to disclose their public contact.
+        """
+        logger.info("Updating security emails to public")
+
+        # Initializes domains that need to be disclosed
+
+        statuses = ["ready", "dns needed"]
```

Comment:
```suggestion
        statuses = [Domain.State.READY, Domain.State.DNS_NEEDED]
```
With an import Domain up top

Reply:
updated - thank you for the suggestion!